Alexa.NET/Request/RequestVerification.cs

```diff
@@ -12,6 +12,7 @@ namespace Alexa.NET.Request
     public static class RequestVerification
     {
         private const int AllowedTimestampToleranceInSeconds = 150;
+        private IDictionary<Uri, X509Certificate2> CachedCerts { get; } = new Dictionary<Uri, X509Certificate2>();
```

Comment:
This would need to be a static property as it's a static class

Reply:
sorry about that bug.  
updated.